src/Server/Diagnostics.idr

```diff
@@ -117,7 +117,7 @@ pwarning (ShadowingGlobalDefs _ ns) =
                              :: reflow "is shadowing"
                              :: punctuate comma (map pretty (forget ns)))
              (forget ns)
-pwarning (Deprecated s) = pure $ pretty "Deprecation warning:" <++> pretty s
+pwarning (Deprecated s n) = pure $ pretty "Deprecation warning:" <++> pretty s <++> pretty n
```

Comment:
We can reuse the definition in the compiler as long as it doesn't also prettyprint the location of the error.

Reply:
You mean reuse the definition of `pretty` for `n : Maybe (FC, Name)`? Just tested the following and looks like it prints the location:
```idris
Server.Diagnostics> :exec Terminal.putDoc $ pretty $ Just ((MkFC (Virtual FC.Interactive) (0,0) (0,0)), UN (Name.Basic "foo"))
((Interactive):1:1--1:1, foo)
```

Reply:
No I mean to copy the compiler code (https://github.com/idris-lang/Idris2/blob/main/src/Idris/Error.idr#L183)

Reply:
Ah nice! Updated.